src/codegen/codegen_c_visitor.cpp

```diff
@@ -2370,7 +2378,9 @@ void CodegenCVisitor::print_coreneuron_includes() {
  * Note that static variables are already initialized to 0. We do the
  * same for some variables to keep same code as neuron.
  */
-void CodegenCVisitor::print_mechanism_global_var_structure(bool wrapper) {
+void CodegenCVisitor::print_mechanism_global_var_structure() {
+    const auto decorator = global_var_struct_type_decorator();
```

Comment:
```suggestion
    const auto qualifier = global_var_struct_type_qualifier();
```
and then adjust everywhere below